concepticondata/conceptlists/Carling-2017-240c.tsv

```diff
@@ -121,8 +121,8 @@ Carling-2017-240c-119	119			lord, boss, proprietor	12260	Social roles and positi
 Carling-2017-240c-120	120	812	SLAVE	slave	12261	Social roles and positions	
 Carling-2017-240c-121	121	255	SERVANT	servant	12262	Social roles and positions	
 Carling-2017-240c-122	122	384	FREEMAN	free man	12263	Social roles and positions	
-Carling-2017-240c-123	123	1128	ORDER	to order, to command	12264	Social roles and positions	
-Carling-2017-240c-124	124	1128	ORDER	to give order	12265	Social roles and positions	
```

Comment:
Why is 1128 removed for 'to give order'?

Reply:
Didn't want to link two things to the same concept, and the first seems more general.